### DISCUSSION TITLE
Embed Script (Forms) / iframe / Form

### DISCUSSION THREAD

**Discussion by backingcorp** [swellstores/community#129] [Help] [answered]
Hi, how is going today? Which is the fastes way to embed forms like forms or other in a contact us page? 
I like to paste code similar to:

// Example for popup

or embed? 

Easy way if I use the Swell  FREE Cloud Version?

Thanks

**Comment by cristianstanescu** [comment, marked as answer]
@backingcorp looking at [the documentation](https://tally.so/help/embed-your-form) for Tally, it looks like any option described would work with the techniques described in our docs [Updating Swell themes](https://developers.swell.is/storefronts/update-swell-themes-with-git#updating-swell-themes).

For example, for [Origin](https://developers.swell.is/storefronts/origin), you might want to customize the [default.vue](https://github.com/swellstores/origin-theme/blob/b5a4edb90521fc8a3c82a794e47e97e46c9ddc41/layouts/default.vue) layout to include the Tally `<script>`.

You could also [customize](https://developers.swell.is/guides/data-model-customization#content-models) the [page content model](https://github.com/swellstores/origin-theme/blob/091acc8176ea53a794f2cb07a1a44f419f0377a6/config/content/page.json#L9-L18) to create a block in the editor that can keep the url to your Tally form. Like it happens for the [large_media url field](https://github.com/swellstores/origin-theme/blob/091acc8176ea53a794f2cb07a1a44f419f0377a6/config/content/large_media.json#L28-L31), as an example. This way, you could embed a form in any of your page pretty easily from the Admin Dashboard UI.
